src/jit/lsra.cpp

```diff
@@ -1755,12 +1756,12 @@ void LinearScan::identifyCandidates()
     unsigned int floatVarCount = 0;
     unsigned int thresholdFPRefCntWtd = 4 * BB_UNITY_WEIGHT;
     unsigned int maybeFPRefCntWtd = 2 * BB_UNITY_WEIGHT;
-#ifdef FEATURE_SIMD
+#if FEATURE_PARTIAL_SIMD_CALLEE_SAVE
     VarSetOps::AssignNoCopy(compiler, largeVectorVars, VarSetOps::MakeEmpty(compiler));
     VarSetOps::AssignNoCopy(compiler, largeVectorCalleeSaveCandidateVars, VarSetOps::MakeEmpty(compiler));
     unsigned int largeVectorVarCount = 0;
     unsigned int thresholdLargeVectorRefCntWtd = 4 * BB_UNITY_WEIGHT;
-#endif // FEATURE_SIMD
+#endif // FEATURE_PARTIAL_SIMD_CALLEE_SAVE
```

Comment:
@sivarv @AndyAyersMS  I usually forget that I'm using markdown in these comments. In the above, I should have said:
there should be little to no difference for raytracer because it does not use `Vector<T>`
and
ConsoleMandel uses `Vector<T>`